packages/react-server/core/ClientController.js

```diff
@@ -650,7 +650,11 @@ class ClientController extends EventEmitter {
 
 			_.forEach(
 				getRootElementAttributes(element),
-				(v, k) => root.setAttribute(k, v)
+				(v, k) => {
+					if (k !== 'tagName') {
+						root.setAttribute(k, v)
+					}
+				}
```

Comment:
Oh, I hadn't gotten far enough... looks like it's being added as an attr there now.  Hmm...